core/test-api/collection/src/main/java/org/opennms/core/collection/test/JUnitCollectorExecutionListener.java

```diff
@@ -154,7 +154,16 @@ public void afterTestMethod(TestContext testContext) throws Exception {
             }
         }
 
-        FileUtils.deleteDirectory(m_snmpRrdDirectory);
+        try {
+            FileUtils.deleteDirectory(m_snmpRrdDirectory);
+        } catch (Exception deletionException) {
+            // Windows is failing tests due to spurious cleanup errors
+            deletionException.printStackTrace();
```

Comment:
Can we log a warning instead of just dumping the stack trace? i.e. "Failed to delete $path during cleanup. Contents may still exist in directory for further tests."

Reply:
Added the logger and now carp the message plus the stacktrace.
I dropped the 2nd sentence of your suggestion, as each test within a test class gets its own unique directory. e.g. **/tmp/FileAnticipator_temp_1560178460050_3785da4a20eb680a**